src/aiodynamo/credentials.py

```diff
@@ -370,17 +419,21 @@ class InstanceMetadataCredentialsV2(MetadataCredentials):
         ).lower()
         == "true"
     )
-    auth_token: str = ""
+    token_session_duration_seconds: int = 5 * 60
```

Comment:
actually, should we set this to max which is 6 hours (6 * 60 * 60)? since now the refresh threshold is like 15 min, it will always be refreshing if it were 5 min

Reply:
that sounds good. should we also check that it's <=6h somewhere to avoid mistakes?